### ISSUE TITLE
USI-Life Support Window flickers

### ISSUE THREAD

**Issue by fat-lobyte** [BobPalmer/USI-LS#218]
In some situations, the Life Support window disappears and reappears again in an approximately one second interval. Each flicker is accompanied by the following exception:

```
ArgumentOutOfRangeException: Argument is out of range.
Parameter name: index
  at System.Collections.Generic.List`1[ProtoCrewMember].get_Item (Int32 index) [0x00000] in <filename unknown>:0 
  at LifeSupport.LifeSupportMonitor.CheckEVAKerbals () [0x00000] in <filename unknown>:0 
  at LifeSupport.LifeSupportMonitor.GenerateWindow () [0x00000] in <filename unknown>:0 
  at LifeSupport.LifeSupportMonitor.OnWindow (Int32 windowId) [0x00000] in <filename unknown>:0 
  at UnityEngine.GUILayout+LayoutedWindow.DoWindow (Int32 windowID) [0x00000] in <filename unknown>:0 
  at UnityEngine.GUI.CallWindowDelegate (UnityEngine.WindowFunction func, Int32 id, UnityEngine.GUISkin _skin, Int32 forceRect, Single width, Single height, UnityEngine.GUIStyle style) [0x00000] in <filename unknown>:0 
```

Full log: [output_log.zip](https://github.com/BobPalmer/USI-LS/files/747363/output_log.zip)
Quicksave: [quicksave.zip](https://github.com/BobPalmer/USI-LS/files/747377/quicksave.zip)

I'm afraid I couldn't make a clean reproduction save, so my main game save is attached. That requires the mods KAS and KIS to load the scene (and NearFutureElectrical for other irrelevent ships), unfortunately.

Screenshot: 
![screenshot22](https://cloud.githubusercontent.com/assets/173609/22544873/e84db624-e936-11e6-9e5b-5c9148244964.png)

**Comment by jd284** [comment]
If in the lines before that you also get "Applying EVA effect", it's probably the same as #212 .

**Comment by fat-lobyte** [comment]
> If in the lines before that you also get "Applying EVA effect"

I do not, and I also don't have any Kerbals on EVA.

**Comment by fat-lobyte** [comment]
However, upon building the current DEVELOPMENT branch with the fixes from PR #213, the exception and the flicker don't occur anymore.

ps.: I love the LS window changes!
